docs/blocks-storage/store-gateway.md

```diff
@@ -283,6 +283,10 @@ blocks_storage:
     # CLI flag: -blocks-storage.s3.endpoint
     [endpoint: <string> | default = ""]
 
+    # S3 region. If unset an S3 API call is made to autodetect it.
```

Comment:
Which S3 API call is made to auto detect region?

Reply:
The Minio client uses this API: https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetBucketLocation.html

Reply:
I see. Do you think it's worth it to be more clean in the doc and just spell out that GetBucketLocation API is called?

Reply:
Yes, done.